sound/pulseaudio/Makefile

```diff
@@ -122,13 +122,15 @@ MESON_ARGS += \
 	-Dx11=disabled \
 	-Dadrian-aec=true \
 	-Dwebrtc-aec=disabled \
-        -Ddoxygen=false
+        -Ddoxygen=false \
+        -Dtcpwrap=false
 
 ifeq ($(BUILD_VARIANT),avahi)
 MESON_ARGS += \
 	-Davahi=enabled \
 	-Dbluez5=enabled \
-	-Ddbus=enabled
+	-Ddbus=enabled \
+	-Dgstreamer=false
```

Comment:
@Ansuel: Gstreamer is already "disabled" at line 109. Please comment for inclusion in https://github.com/openwrt/packages/pull/22541 on what you thing should be done about this issue. Kind of stuck here.